packages/samples/react/src/scenarios/appointment-form/PersonalInformationForm.tsx

```diff
@@ -41,6 +50,10 @@ export function PersonalInformationForm() {
 				<Field name="salutation">
 					{({ field }: FieldProps<FormValues['salutation']>) => (
 						<KolSelect
+							onKeyDown={(event) => {
+								if (event.key === 'Tab') void form.setFieldTouched('salutation', true);
```

Comment:
Wir sollten hier stattdessen das `blur` event verwenden.

Die aktuelle Lösung wird nur funktionieren, wenn tatsächlich `<tab>` gedrückt wird, aber nicht, wenn der User z.B. in das nächste Feld klickt.